_pytest/python.py

```diff
@@ -1014,16 +1014,24 @@ def _showfixtures_main(config, session):
     fm = session._fixturemanager
 
     available = []
+    seen = set()
+
     for argname, fixturedefs in fm._arg2fixturedefs.items():
         assert fixturedefs is not None
         if not fixturedefs:
             continue
         for fixturedef in fixturedefs:
             loc = getlocation(fixturedef.func, curdir)
+            fixture_argname = fixturedef.argname
+            if ':' in fixture_argname:
```

Comment:
When it's a invocation-scoped fixture. For example, defining `monkeypatch` as invocation-scoped will generate 4 fixture-defs in different scopes: `"monkeypatch:function"`, `"monkeypatch:class"`, `"monkeypatch:module"` and `"monkeypatch:session"`.